operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -291,7 +291,12 @@ type CollectorContainerSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
 	ImageFlavor *CollectorImageFlavor `json:"imageFlavor,omitempty"`
 
+	// Set this to 'true' to prevent translation of the collection method, e.g. to enforce using EBPF as an exception.
```

Comment:
I think that the documentation could be improved a bit. The CSV is customer facing, but "translation of the collection method" is something internal to the operator.